docs/proposals/block-storage-time-series-deletion.md

```diff
@@ -0,0 +1,272 @@
+---
+title: "Time Series Deletion from Blocks Storage"
+linkTitle: "Time Series Deletion from Blocks Storage"
+weight: 1
+slug: block-storage-time-series-deletion
+---
+
+- Author: [Ilan Gofman](https://github.com/ilangofman)
+- Date: June 2021
+- Status: Proposal
+
+## Problem
+
+Currently, Cortex only implements a time series deletion API for chunk storage.  We present a design for implementing time series deletion with block storage. We would like to have the same API for deleting series as currently implemented in Prometheus and in Cortex with chunk storage. 
+
+
+This can be very important for users to have as confidential or accidental data might have been incorrectly pushed and needs to be removed. 
+
+## Related works
+
+As previously mentioned, the deletion feature is already implemented with chunk storage. The main functionality is implemented through the purger service. It accepts requests for deletion and processes them.  At first, when a deletion request is made, a tombstone is created. This is used to filter out the data for queries. After some time, a deletion plan is executed where the data is permanently removed from chunk storage. 
+
+Can find more info here:
+
+- [Cortex documentation for chunk store deletion](https://cortexmetrics.io/docs/guides/deleting-series/)
+- [Chunk deletion proposal](https://docs.google.com/document/d/1PeKwP3aGo3xVrR-2qJdoFdzTJxT8FcAbLm2ew_6UQyQ/edit)
+
+
+
+## Background on current storage
+
+With a block-storage configuration, Cortex stores data that could be potentially deleted by a user in:
+
+- Object store (GCS, S3, etc..) for long term storage of blocks
+- Ingestors for more recent data that should be eventually transferred to the object store
```

Comment:
```suggestion
- Ingesters for more recent data that should be eventually transferred to the object store
```

Reply:
There's also other instances of "ingestor" later in the document.

Reply:
Good catch, thanks!